Looks like there is a small CI failures

```
> @openai/codex@0.1.2504221401 format /home/runner/work/codex/codex/codex-cli
> prettier --check src tests

Checking formatting...
[warn] src/text-buffer.ts
[warn] Code style issues found in the above file. Forgot to run Prettier?
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

could you take a look and then I will merge. Thank you for making your first contribution to codex!

--- a/codex-cli/src/text-buffer.ts
+++ b/codex-cli/src/text-buffer.ts
@@ -34,6 +34,10 @@ function clamp(v: number, min: number, max: number): number {
  * ---------------------------------------------------------------------- */
 
 function toCodePoints(str: string): Array<string> {
+  if (typeof Intl !== 'undefined' && 'Segmenter' in Intl) {
+    const seg = new Intl.Segmenter()
+    return [...seg.segment(str)].map((seg) => seg.segment)
+  }
   // [...str] or Array.from both iterate by UTF‑32 code point, handling
   // surrogate pairs correctly.
   return Array.from(str);

--- a/codex-cli/src/text-buffer.ts
+++ b/codex-cli/src/text-buffer.ts
@@ -34,9 +34,9 @@ function clamp(v: number, min: number, max: number): number {
  * ---------------------------------------------------------------------- */
 
 function toCodePoints(str: string): Array<string> {
-  if (typeof Intl !== 'undefined' && 'Segmenter' in Intl) {
-    const seg = new Intl.Segmenter()
-    return [...seg.segment(str)].map((seg) => seg.segment)
+  if (typeof Intl !== "undefined" && "Segmenter" in Intl) {
+    const seg = new Intl.Segmenter();
+    return [...seg.segment(str)].map((seg) => seg.segment);
   }
   // [...str] or Array.from both iterate by UTF‑32 code point, handling
   // surrogate pairs correctly.

